Condition expressions/If statement/task.md

```diff
@@ -1,7 +1,20 @@
 ## If statement
 
-The `if` keyword is used to form a conditional statement that executes some specified code after checking if its expression is `True` . Python uses indentation to define code blocks.  
-  
+Compound statements in Python contain (groups of) other statements; they affect or control 
+the execution of those other statements in some way. The `if`, `while` and `for` statements
```

Comment:
We are nott actually talking in tterms of control flow here in the lesson, so we probably need o loose the mention of the loops and simplify the description